### DISCUSSION TITLE
How to make eww window overlap

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [elkowar/eww#897] [Q&A]
![image](https://github.com/elkowar/eww/assets/61395246/f3492a76-4626-45d7-8792-e5a1a97f4975)

How can i make eww windows overlap ?  I wish for the powermenu window to take the full screen
```yuck

(defwindow bar
    :monitor 0
    :geometry (geometry :x "0%"
        :y "5px"
        :width "90%"
        :height "4%"
        :anchor "top center")
    :exclusive true
    :namespace "eww"
(bar))
....

(defwindow powermenu
:monitor 0
:focusable true
  :stacking "overlay"
  :geometry (geometry :width "100%" :height "100%")
  (powermenu))

```

**Comment by mantacid** [comment]
have you tried the `wm-ignore` property?

**Reply by LamprosPitsillos** [reply]
I'm on Wayland

**Reply by mantacid** [reply]
oh, then maybe it's the fact that you have `:exclusive true` in your config. Have you tried setting it to false?

**Reply by LamprosPitsillos** [reply]
Sorry maybe i wasn't clear enough, tge bar is supposed to be like that, with the `exclude`. The Power menu is the one bothering me. I want it to take ALL the screen but it stops at the bar bottom (i imagine due to the `exlusive`).
I tried setting `overlay` to the powermenu hoping it would ve *on top* of the bar bbut it didn't work

**Reply by mantacid** [reply]
Oh sorry about that. Maybe look into the Wayland layersurface namespace documentation, it seems like there’s a parameter that might affect what your trying to accomplish. https://wayland.app/protocols/wlr-layer-shell-unstable-v1

**Comment by end-4** [comment]
you're on hyprland right? you can [create a custom reserved area](http://wiki.hyprland.org/Configuring/Monitors/#custom-reserved-area) and remove `:exclusive true` from your bar. then the power menu will be fullscreen
